modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -481,6 +488,43 @@ default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
 |===
 
+
+[discrete]
+[id="installation-vsphere-installer-infra-static-ip-nodes_{context}"]
+== Specifying static IP addresses for vSphere nodes
+
+You can provision bootstrap, control plane, and compute nodes to use static IP addresses, so that you can better configure your network to meet the security demands of your organization. To configure this environment, you must provide values to the `platform.vsphere.hosts.role` parameter in the `install-config.yaml` file.
+
+:FeatureName: Specifying static IP addresses for vSphere nodes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+By default, the installation program is configured to use the Dynamic Host Configuration Protocol (DHCP) for the network, but this network has limited configurable capabilities.
+
+The following example shows a `compute` network definition specified for the `role` parameter:
+
+[source,yaml]
+----
+---
+platform:
+  vsphere:
+    hosts:
+    - role: compute <1>
+    networkDevice:
+      ipAddrs:
+      - 192.168.204.10/24 <2>
+      gateway: 192.168.204.1 <3>
+      nameservers:
+      - 192.168.204.1 <4>
+---
+----
+<1> Valid network defintion values include `bootstrap`, control-plane`, and `compute`.
+<2> Lists IPv4, IPv6, or both IP addresses that installation program passes to the network interface. Both IP families must use the same network interface for the default network. 
+<3> The default gateway for the network interface. 
+<4> Lists up to 3 DNS servers that can resolve provided provided IP addresses by a process known as DNS resolution.
```

Comment:
SME query: I see `you must verify that this IP address has a valid internal domain name system (DNS) registration.` in the [VMware documentation](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.install.doc/GUID-24D34C53-B00E-47B7-92A7-6B0155DF6889.html). Do I need to state this?